presto-main/src/test/java/io/prestosql/sql/query/TestAggregationOverJoin.java

```diff
@@ -15,24 +15,25 @@
 
 import org.testng.annotations.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class TestAggregationOverJoin
 {
     @Test
     public void test()
     {
         // https://github.com/prestodb/presto/issues/10592
-        try (QueryAssertions queryAssertions = new QueryAssertions()) {
-            queryAssertions
-                    .assertQuery(
+        try (QueryAssertions assertions = new QueryAssertions()) {
+            assertThat(assertions.query(
                             "WITH " +
                                     "    t (a, b) AS (VALUES (1, 'a'), (1, 'b')), " +
                                     "    u (a) AS (VALUES 1) " +
                                     "SELECT DISTINCT v.a " +
                                     "FROM ( " +
                                     "    SELECT DISTINCT a, b " +
                                     "    FROM t) v " +
-                                    "LEFT JOIN u on v.a = u.a",
-                            "VALUES 1");
+                                    "LEFT JOIN u on v.a = u.a"))
+                    .matches("VALUES 1");
```

Comment:
.matches suggests some "pattern matching", while here it's equality (disregaring orders)
any better name? `returns`?

Reply:
Hmmm... that's a very narrow interpretation of "matches". That word means, generally, to "be equal to" or "be similar to". I think it conveys the intent closely enough: "the results of the current query match the results of the given query". 

On the other hand, "returns" also suggests it returns that exact string value, so it's not necessarily a better choice.